scripts/backups/init-backups.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+#
+# SneezyMUD Backup Setup - Simple automated backups for SneezyMUD
+#
+set -e
+
+info() { echo "→ $1"; }
+success() { echo "✓ $1"; }
+error() { echo "✗ $1" >&2; exit 1; }
+
+if [[ "${1:-}" == "--undo" ]]; then
+    UNDO_MODE=true
+elif [[ "${1:-}" == "--help" ]] || [[ "${1:-}" == "-h" ]]; then
+    echo "SneezyMUD Backup Setup"
+    echo "Usage: sudo $0 [--undo]"
+    exit 0
+else
+    UNDO_MODE=false
+fi
+
+[[ $EUID -eq 0 ]] || error "Must run with sudo"
+command -v docker &> /dev/null || error "Requires Docker"
+systemctl is-active --quiet docker || error "Docker not running"
+
+get_config() {
+    BACKUPS_USER="${BACKUPS_USER:-sneezy-backups}"
+    BACKUPS_DIR="${BACKUPS_DIR:-/opt/backups/sneezy}"
```

Comment:
What's the plan for moving these off-machine?

Reply:
These backups are also what the `backups-status` channel on the Discord is referring to

Reply:
Oh wow! Retaining _all_ backups without deleting any old ones? I guess GitHub is OK with that, apparently... How big is the repo by now?

In Old Times™, we used a Google Drive account (5GB), which got full every now and then, so I had to go there and manually delete stuff.